mne/bem.py

```diff
@@ -2187,3 +2187,59 @@ def check_seghead(surf_path=op.join(subj_path, 'surf')):
             incomplete=incomplete, extra=msg)
         write_bem_surfaces(dec_fname, dec_surf, overwrite=overwrite)
     logger.info('[done]')
+
+
+def distance_to_bem(
+    pos: np.array, bem: ConductorModel, trans: Transform = None
+) -> np.array:
+    """Calculate the distance of postions to inner skull surface.
+
+    Parameters
+    ----------
+    pos : array, shape = (3,) | (n, 3)
+        Position(s) in m, in head coordinates.
+    bem : instance of ConductorModel
+        Conductor model.
+    trans : array, shape = (4, 4) | instance of Transform
```

Comment:
you should use the docdict entry for trans. As for example:

https://github.com/mne-tools/mne-python/blob/main/mne/viz/_3d.py#L433

Reply:
also it should be clear that None is allowed

Reply:
I believe I have fixed this in the last commit. Not familiar with docdict however.